### PR TITLE
feat: Message list supports top to bottom layout #212

### LINKED ISSUE #212: Top-to-bottom message list
The message list scrolls from bottom to top by default, the latest message appears at the bottom of the page.

Add an input to the message list to reverse the order: the latest message is on the top of the page, and scrolling works top to bottom direction.

### PR DESCRIPTION
**Pull request by szuperaz**
closes #212

### CODE REVIEW

**Comment by szuperaz** [comment]
Note to self: update CSS version before merge

**Comment by szuperaz** [comment]
> Note to self: update CSS version before merge

Oh no, this doesn't count as unresolved conversation?

File: projects/stream-chat-angular/src/lib/icon/icon.component.html [hunk 1]
```diff
@@ -254,3 +254,32 @@
     fill="black"
   />
 </svg>
+
```
Comment (szuperaz) [inline projects/stream-chat-angular/src/lib/icon/icon.component.html:257]: Maybe this?

Comment (szuperaz) [reply projects/stream-chat-angular/src/lib/icon/icon.component.html:257]: Note to self: update CSS version before merge

**Review by petyosi — APPROVED** [review]

**Pull request merged**